src/Durable/Commands/SetDurableCustomStatusCommand.cs

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#pragma warning disable 1591 // Missing XML comment for publicly visible type or member 'member'
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable.Commands
+{
+    using System.Collections;
+    using System.Management.Automation;
+
+    [Cmdlet("Set", "DurableCustomStatus")]
+    public class SetDurableCustomStatusCommand : PSCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            Position = 0,
```

Comment:
Do we need to assign a `Position` for this parameter? I took a look at the other Durable cmdlets and this is the only one with a Positional parameter.

Reply:
I'm trying to be very careful with adding positional parameters in general, but in this case I believe this is justified. The cmdlet takes just one parameter, and the purpose of this parameter is very obvious, so I want to allow users omit the parameter name: `Set-DurableCustomStatus "my status"`.

I think we can carefully allow _some_ positional parameters for _some_ other durable-related cmdlets, but we can do this later.

Reply:
Sounds good @AnatoliB, thank you.